components/home/Welcome.js

```diff
@@ -18,10 +18,13 @@ class Welcome extends Component {
               <TopNavigation title='Welcome' alignment='center'/>
               <Divider/>
               <Layout style={styles.centeredContainer}>
-                <Button style={styles.styledButton} onPress={() => navigateTo('Survey')}>Start New Report</Button>
-              </Layout>
+                    <View style={styles.btnContainer}>
+                        <TouchableOpacity style={styles.styledButton} activeOpacity = { .7 } onPress={() => navigateTo('Survey')}>
+                            <Text style={styles.btnText}>Quick Start</Text>
+                        </TouchableOpacity>
+                    </View>
+               </Layout>
               <Layout style={styles.bottomBar}>
-                <TouchableOpacity onPress={()=>Linking.openURL('https://forms.gle/ho3cZNyoaFArNNN79')}><Text style={{color: 'blue'}}>Submit Feedback</Text></TouchableOpacity>
                 <Text style={styles.bottomBarText}>
                   {"Built by students at Olin College of Engineering 2020"}
```

Comment:
Should be "Built by students at Olin College of Engineering 2020-2021"?

Reply:
Great that can be captured in another card for this sprint!